pkg/api/customization/kontainerdriver/actionhandler.go

```diff
@@ -137,12 +137,12 @@ func (lh ListHandler) LinkHandler(apiContext *types.APIContext, next types.Reque
 	var targetImages []string
 	switch apiContext.ID {
 	case linuxImages:
-		targetImages, err = image.GetImages(systemCatalogChartPath, []string{}, rkeSysImages, image.Linux)
+		targetImages, err = image.GetImages(systemCatalogChartPath, []string{}, []string{}, rkeSysImages, image.Linux)
```

Comment:
this may have been me, but I believe this was a mistake. Should be rkeSysimages still.

Reply:
This is still `rkeSysImages`. Current func signature is `GetImages(systemChartPath string, k3sUpgradeImages, imagesFromArgs []string, rkeSystemImages map[string]v3.RKESystemImages, osType OSType) ([]string, error)` 
arg[3] is and should be rkeSysImage ✨